news/2024/2024-11-17-kel-lan-tournament-3.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: post
+title: KEL LAN Tournament 3
+date: 2024-11-17 12:00:00 +0000
+---
+
+Calling all Circle Clickers to Poland! For the third iteration of KEL Tournament, a major in-person tournament with special guests!
+
+![](/wiki/shared/news/2024-11-17-kel-lan-tournament-3/banner.jpg)
+
+KEL3 is a **2-day LAN** tournament at **Kinguin Esports Lounge in Gdańsk, Poland**, standing as the largest event of its kind in the country, where top-tier competition meets an incredible community experience
+
+Depending on amount of **accepted invites** by **special guests**, only **certain amount** of people will **qualify** to the tournament.
```

Comment:
that's a lot of bolded words?

Reply:
I can fix that if it's really neccesary.